skins/wmde19/src/components/layout/Header.vue

```diff
@@ -1,11 +1,56 @@
 <template>
-	<div class="header">
-		Header
+	<div class="navbar is-fixed-top has-background-white">
+		<div class="container">
+			<div class="navbar-brand">
+				<a class="navbar-item">
+					<img :src="imagePath + '/images/logo-horizontal-wikimedia.svg'" alt="Wikimedia Deutschland" width="144" height="29">
+				</a>
+				<a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false">
+					<span aria-hidden="true"></span>
+					<span aria-hidden="true"></span>
+					<span aria-hidden="true"></span>
+				</a>
+			</div>
+
+			<div class="navbar-menu" id="navMenu">
+				<div class="navbar-start">
```

Comment:
Especially, if we later want to have a feature that highlights the current page. Although that one can be achieved with css classes instead of using `v-if` in the loop.